### PR TITLE
Feature/ADF-213/Configure delete and upload endpoints

### PR DESCRIPTION
**Pull request by shpran**
**Relates to:** _[ADF-213](https://oat-sa.atlassian.net/browse/ADF-213)_

### CODE REVIEW

**Review by chinnu-focaloid — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

File: manifest.php [hunk 1]
```diff
@@ -175,11 +175,6 @@
             AccessRule::GRANT,
             TaoItemsRoles::ITEM_CONTENT_CREATOR,
             ['ext' => 'taoItems', 'mod' => 'Items', 'act' => 'authoring'],
-        ],
-        [
```
Comment (gabrielfs7) [inline manifest.php:179]: why was this removed?

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-shpran.playground.kitchen.it.taocloud.org:41481/
